homeassistant/components/alarmdecoder/binary_sensor.py

```diff
@@ -52,10 +52,9 @@ async def async_setup_entry(
         device = AlarmDecoderBinarySensor(
```

Comment:
Maybe rename this to `entity`?

Reply:
done in https://github.com/home-assistant/core/pull/40037/commits/e4e79b49494a68581169209ca8849d6579e06521